.github/dockerfiles/ov_build/ubuntu_22_04_android_arm64/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM openvinogithubactions.azurecr.io/dockerhub/ubuntu:22.04
+
+USER root
+
+# APT configuration
+RUN echo 'Acquire::Retries "10";' > /etc/apt/apt.conf && \
+    echo 'APT::Get::Assume-Yes "true";' >> /etc/apt/apt.conf && \
+    echo 'APT::Get::Fix-Broken "true";' >> /etc/apt/apt.conf && \
+    echo 'APT::Get::no-install-recommends "true";' >> /etc/apt/apt.conf
+
+ENV DEBIAN_FRONTEND="noninteractive" \
+    TZ="Europe/London"
+
+RUN apt update && \
+    apt install software-properties-common git ca-certificates && \
+    add-apt-repository --yes --no-update ppa:git-core/ppa && \
+    apt update && \
+    apt install \
+        scons \
+        wget \
+        ninja-build \
+        build-essential \
+        python3-pip && \
+    # vcpkg requires cmake 3.19 or later
+    python3 -m pip install -U pip cmake~=3.28.0 && \
+    # vcpkg's tool dependencies
+    apt install curl zip unzip tar && \
+    # vcpkg 'python3' port dependencies
+    apt install autoconf libtool autoconf-archive && \
+    # vcpkg tree of dependencies require extra packages
+    apt install pkgconf linux-libc-dev && \
+    apt --no-install-recommends install default-jdk && \
```

Comment:
If combined into one `apt install`, `apt` cannot determine the matching versions for all the packages and breaks.